core-kotlin-modules/core-kotlin-strings-2/src/test/kotlin/com/baeldung/digits/CountDigitsUnitTest.kt

```diff
@@ -0,0 +1,71 @@
+package com.baeldung.digits
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class CountDigitsUnitTest {
+
+    private fun Int.countDigits() = when(this) {
+        0 -> 1
+        else -> kotlin.math.log10(kotlin.math.abs(toDouble())).toInt() + 1
```

Comment:
Let's import the `log10` and `abs` functions at the top, so that this line can read just

```kotlin
else -> log10( abs( toDouble() ) ).toInt() + 1
```